src/openrct2/CmdlineSprite.cpp

```diff
@@ -543,9 +543,10 @@ int32_t cmdline_for_sprite(const char** argv, int32_t argc)
 
         const char* spriteFilePath = argv[1];
 
-        spriteFile.Header.num_entries = 0;
-        spriteFile.Header.total_size = 0;
-        sprite_file_save(spriteFilePath);
+        SpriteFile sFile;
+        sFile.Header.num_entries = 0;
+        sFile.Header.total_size = 0;
+        sFile.Save(spriteFilePath);
```

Comment:
(On commit https://github.com/OpenRCT2/OpenRCT2/pull/13590/commits/3b39ab2299e88ae7cee242006590ab3ec0415b8f)

Shouldn't this be using the `static` variable `spriteFile`?

Reply:
no it is localised to just this branch of the if no need to be the global